hyperdash/experiment.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+from .client import HDClient
+from .monitor import monitor, get_logger
+from .io_buffer import IOBuffer
+from .server_manager import ServerManagerHTTP
+from .hyper_dash import HyperDash
+
+import logging
+import sys
+import uuid
+import threading
+
+# Python 2/3 compatibility
+__metaclass__ = type
+
+class ExperimentRunner:
```

Comment:
I'm not quite following what this is for

Reply:
Passed into the HyperDash object instead of CodeRunner. Gives the same callbacks. I may refactor HyperDash.run() to not need this.